documentation/docs/guides/masonry-layout.md

```diff
@@ -0,0 +1,85 @@
+---
+id: masonry
+title: Masonry Layout
+---
+
+Masonry Layout allows you to create a grid of items with different heights. It is a great way to display a collection of images with different sizes.
+
+<div align="center">
+<img src="https://user-images.githubusercontent.com/7811728/188055598-41f5c961-0dd0-4bb9-bc6e-22d78596a036.png" height="500"/>
+</div>
+
+To get started, import `MasonryFlashList` from `@shopify/flash-list` and use it just like you would use `FlashList`:
+
+**Note:** If you want `MasonryFlashList` to optimize item arrangement, enable `optimizeItemArrangement` and pass a valid [`overrideItemLayout`](../fundamentals/usage.md#overrideitemlayout) function.
+
+## Unsupported Props
+
+There are some props that `MasonryFlashList` does not support when compared to `FlashList`. These are:
+
+- [`horizontal`](../fundamentals/usage.md#horizontal)
+- [`inverted`](../fundamentals/usage.md#inverted)
+- [`initialScrollIndex`](../fundamentals/usage.md#initialscrollindex)
+- [`viewabilityConfigCallbackPairs`](../fundamentals/usage.md#viewabilityconfigcallbackpairs)
+- [`onBlankArea`](../fundamentals/usage.md#onblankarea)
+
+## Additional Props
+
+There are some additional props that `MasonryFlashList` supports when compared to `FlashList`. These are:
```

Comment:
```suggestion
`MasonryFlashList` supports these additional props on top of `FlashList`:
```

`when compared to` sounds odd to me here and we can make the wording more concise.